### DISCUSSION TITLE
Does Quarto (re)install its own version of Pandoc?

### DISCUSSION THREAD

**Discussion by mgiugliano** [quarto-dev/quarto-cli#1642] [Q&A] [answered]
I wonder whether I can safely play and explore Quarto, without affecting my existing Pandoc installation and experimenting.

I am under macOs and I install everything with Homebrew (cask). Should I worry for conflicts, for multiple versions, etc.?

**Comment by cscheid** [comment, marked as answer]
Quarto bundles its own version of pandoc, which shouldn't conflict with anything in your filesystem.

**Reply by jjallaire** [reply]
Also, if you want to access the version of pandoc bundled with Quarto you can do so with:

```bash
quarto pandoc <any-pandoc-args>
```

For example:

```bash
quarto pandoc --version
quarto pandoc --list-output-formats
quarto pandoc foo.md --to gfm --citeproc
```

**Reply by mgiugliano** [reply]
Thank you both!

**Reply by jhchou** [reply]
> Quarto bundles its own version of pandoc, which shouldn't conflict with anything in your filesystem.

While the following might NOT actually qualify as a "conflict", but I just posted on the RStudio community forums my experience with Pandoc failures after upgrading RStudio IDE: https://community.rstudio.com/t/rstudio-update-causes-r-command-line-failure-to-find-pandoc-which-has-been-moved-to-quarto-folder/150846

It's not a "conflict", and I'm not sure how command line executed R using `rmarkdown` originally knew where to find Pandoc, but it couldn't find it any more after the change in location.

**Comment by jjallaire** [comment]
Packages (including **rmarkdown**) generally check the `RSTUDIO_PANDOC` environment variable to determine the location of Pandoc bundled with RStudio, so a change of location shouldn't cause a problem (assuming this variable is being properly checked). Not sure what is going on in the scenario you are describing.
